manager/controllers/utils.go

```diff
@@ -20,3 +24,28 @@ func LogAndRespNotFound(c *gin.Context, err error, respMsg string) {
 	utils.Log("err", err.Error()).Warn(respMsg)
 	c.JSON(http.StatusNotFound, ErrorResponse{respMsg})
 }
+
+func ApplySort(c *gin.Context, tx *gorm.DB, allowedFields ...string) (*gorm.DB, error) {
+	query := c.DefaultQuery("sort", "id")
```

Comment:
We discussed this with jdostal, decided that copying the behavior of vulnerabilities was the best idea. If we have used `public_date` as a default sort, it would have to descending, and if we would have used other - text fields, they would have to be ascending. 

This way we have common sort method, which guarantess stable ordering, and allows sorting on different fields.

Reply:
Ok.